dashboards/CockroachDB/CockroachDB-Details-Storage.json

```diff
@@ -427,22 +461,22 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(sum(livebytes{job=\"cockroachdb\",cluster=\"$cluster\",instance=~\"$node\"}) by (instance))",
+          "expr": "sum(sum(livebytes{dcos_component_name=\"CockroachDB\",host=~\"$node\"}) by (host))",
           "format": "time_series",
           "hide": false,
           "interval": "",
           "intervalFactor": 2,
-          "legendFormat": "Live",
+          "legendFormat": "DC/OS",
```

Comment:
This is the total bytes used for storing keys and values. The label `DC/OS` is a bit misleading, making it sound like it is measuring something for the entire cluster.  I would comprehend `user` and `system` more easily as labels here, or even `user` and `internal`.